### ISSUE TITLE
Expose the event types so it could be used with svelte typescript

### ISSUE THREAD

**Issue by joseDaKing** [mhmd-22/ezgesture#1]

**Comment by qurafi** [comment]
Thank you for your suggestion. I will work on that on the next release.

**Comment by qurafi** [comment]
Event types now are shipped in the latest 1.1.4 version.

First make sure you reference types in your tsconfig file:
```javascript 
"include": [
    "node_modules/ezgesture/types/*"
],
```

Then you can use you the types globally in svelte without importing them:
```svelte
<button on:ezgdragstart="{onDragStart}"></button>
```
Here in markup It will automatically check function type. but you have to do it manually in the function definition:
```typescript
function onDragStart(event: EZGDragEvent) {
  console.log(event.detail.startX);
}
```

For DOM Events. it will automatically do types for you:
```javascript
document.body.addEventListener("ezgdragstart", ev => {
   console.log(ev.detail.startX)
})
```


List of available interfaces(Available globally):
```javascript
EZGDragEvent // for all drag events

EZGPinchStartEvent
EZGPinchMoveEvent
EZGPinchEndEvent 

EZGLongPressEvent
```
\* Replace Event with Detail to get access to parameter types
